gulp/modules/Plugins.js

```diff
@@ -7,17 +7,18 @@ var Plugins = function() {
 	this.del = require('del');
 	this.autoprefixer = require('gulp-autoprefixer');
 	this.batch = require('gulp-batch');
+	// TODO: remove
```

Comment:
Can you expand this comment a bit more? Which plugins should be removed, just `changed`?

Reply:
I just removed it and the batch dependency as they aren't used anywhere

Reply:
I just went ahead and removed it, thanks!